app/uk/gov/hmrc/plasticpackagingtaxregistrationfrontend/controllers/HelloWorldController.scala

```diff
@@ -10,11 +26,11 @@ import uk.gov.hmrc.plasticpackagingtaxregistrationfrontend.views.html.hello_worl
 import scala.concurrent.Future
 
 @Singleton
-class HelloWorldController @Inject()(
+class HelloWorldController @Inject() (
   appConfig: AppConfig,
   mcc: MessagesControllerComponents,
-  hello_world: hello_world)
-    extends FrontendController(mcc) {
+  hello_world: hello_world
+) extends FrontendController(mcc) {
 
   implicit val config: AppConfig = appConfig
```

Comment:
The same here

Reply:
This is awkward... This should have been injected as implicit. There is no need to do it in 2 steps.